content/en/developers/infrastructure/libraries/index.md

```diff
@@ -15,3 +15,19 @@ aliases:
 ---
 
 {{< beta-warning >}}
+
+## Zondax 
+
+Zondax provides the [Filecoin Solidity npm package](#filecoin-solidity) and the [FEVM Precompiles library](#fevm-precompiles).
+
+## Filecoin.sol
+
+{{< alert >}}
+The Filecoin Solidity project is [currently in beta](https://docs.zondax.ch/fevm/filecoin-solidity#disclaimer-%EF%B8%8F%EF%B8%8F).
+{{< /alert >}}
+
+_Filecoin.sol_ provides a set of libraries that allows Solidity smart contracts to seamlessly call built-in actors methods. Additionally, a set of mock libraries that respond to specific scenarios is available. The scenarios are based on the received parameters instead of real calls. For further information, including information on how to use the package, see the [official documentation](https://docs.zondax.ch/fevm/filecoin-solidity/).
```

Comment:
1. Let's be clear that filecoin.solidity allows calling exported built-in actor methods, not all of them.  Some community devs complained to me that filecoin.solidity only provides a few built-in actors calls which is a misunderstanding. 
2. mock version of the libraries is not recommended to use anymore. It was only for supporting FVM hack in wallaby since there was no real deal data. 
3. we should also provide link to their [GitHub](https://github.com/Zondax/filecoin-solidity).

Reply:
@longfeiWan9 

1. Got it, added copy for this
1. Got it, removed mention of mock version
1. Got it, added link